scripts/rebuild-testbench.sh

```diff
@@ -0,0 +1,60 @@
+#!/bin/bash
+# SPDX-License-Identifier: BSD-3-Clause
+# Copyright(c) 2020, Mohana Datta Yelugoti
+
+# fail on any errors
+set -e
+
+print_usage()
+{
+    cat <<EOFUSAGE
+usage: $0 [-f]
+       -f Build testbench with compiler provided by fuzzer
+          (default path: $HOME/sof/work/AFL/afl-gcc)
+EOFUSAGE
+}
+
+rebuild_testbench()
+{
+    cd "$BUILD_TESTBENCH_DIR"
+
+    rm -rf build_testbench
+
+    mkdir build_testbench
+    cd build_testbench
```

Comment:
These can be avoided with CMake 3.13 super convenient `-B` option.
https://cmake.org/cmake/help/v3.13/release/3.13.html#command-line

@jajanusz any plan to raise the minimum CMake required version?